src/azure-cli/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -395,8 +455,14 @@ def resume_protection(cmd, client, resource_group_name, vault_name, container_na
     return None
 
 
-def restore_azure_wl(cmd, client, resource_group_name, vault_name, recovery_config):
-    return custom_wl.restore_azure_wl(cmd, client, resource_group_name, vault_name, recovery_config)
+def restore_azure_wl(cmd, client, resource_group_name, vault_name, recovery_config, rehydration_duration=15,
+                     rehydration_priority=None):
+
+    if rehydration_duration < 10 or rehydration_duration > 30:
+        raise InvalidArgumentValueError('--rehydration-duration must have a value between 10 and 30.')
```

Comment:
same as above. Let's check with someone if this is fine.

Reply:
Same reply as above.